io.catenax.pcf/6.0.0/Pcf.ttl

```diff
@@ -0,0 +1,843 @@
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-01-01T00:00:01Z"^^xsd:dateTime .
+
+:validityPeriodEnd a samm:Property ;
+   samm:preferredName "Validity Period End"@en ;
+   samm:description "Optional: End of interval during which the product (carbon) footprint is declared as valid as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-12-31T23:59:59Z"^^xsd:dateTime .
+
+:comment a samm:Property ;
+   samm:preferredName "Comment"@en ;
+   samm:description "Optional: Additional information and instructions related to the calculation of the product (carbon) footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Comment for version 42." .
+
+:companyName a samm:Property ;
+   samm:preferredName "Company Name"@en ;
+   samm:description "Mandatory: Name of the product (carbon) footprint data owner as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :NonEmptyStringTrait ;
+   samm:exampleValue "My Corp" .
+
+:companyIds a samm:Property ;
+   samm:preferredName "Company IDs"@en ;
+   samm:description "Mandatory: Non-empty set of Uniform Resource Names (URN). Each value is supposed to uniquely identify the product (carbon) footprint data owner as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.1.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. For Catena-X Industry Core compliance the set of URNs must contain at least the Business Partner Number Legal Entity (BPNL) in the specified format urn:bpn:id:BPNL[a-zA-Z0-9]{12}. "@en ;
+   samm:see <ext-bpnl:bpnlProperty> ;
```

Comment:
see value?

Reply:
After alignment with the Catena-X Industry Core team we want to express the semantic compatibility between several properties in the PCF data model and the Industry Core specification. For compliance with the WBCSD (World Business Council for Sustainable Development)/ PACT standard, the PCF data model allows the declaration of several company identifiers. With regards to the Catena-X Industry Core at least one of these company identifiers must be the BPNL (Business Partner Number Legal). Didn't understand the question fully, is there anything wrong with the way this context is implemented? Would be helpful if you could provide more details what correction is expected.

Reply:
@agg3fe : any advise from you? as already mentioned an example would be helpful

Reply:
@SMaierTSI sorry for the confusion. what i mean to say is, inside 'see' element, we usually put some https link, but here you have put '<ext-bpnl:bpnlProperty>'. So is this correct from your side or is this by mistake?

Reply:
Hi @SMaierTSI and @agg3fe, seems correct to me, probably it would be better to use a fully qualified urn instead of the local namespace in the field: 

`samm:see <urn:samm:io.catenax.shared.business_partner_number:2.0.0#bpnlProperty> ;` 

same applies to line 133

Reply:
@SMaierTSI Ok. Perfectly fine for me. I will check for MS2 today and do the MS3 in Monday meeting.

Reply:
@jacewski-bosch , @agg3fe : replaced as suggested. However, for successfull validation it seems there is a ":" needed:

`samm:see <urn:samm:io.catenax.shared.business_partner_number:2.0.0#:bpnlProperty> ;`

Otherwise validation fails.